docs/release-notes/2.21.0.md

```diff
@@ -0,0 +1,53 @@
+# Percona Monitoring and Management 2.21.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">August 24, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## New Features
+
+- [PMM-8158](https://jira.percona.com/browse/PMM-8158): Backup Management: Delete artifact from storage — When deleting a backup from Backup Management, a new option lets you also delete the data file from storage.
+- [PMM-8156](https://jira.percona.com/browse/PMM-8156): Backup Management: Retention — You can now define how many of the most recent scheduled backups to keep.
+- [PMM-8214](https://jira.percona.com/browse/PMM-8214): Ability to collect Kubernetes cluster metrics — Collection only–metrics are not currently presented on any PMM dashboard.
```

Comment:
`only–metrics` this looks weird, I have corrected additional text in Jira to `only, metrics`

Reply:
Thanks, it's better now, seen in context.